Examples/Scripts/FreeSurferPipelineBatch-long.sh

```diff
@@ -55,18 +79,14 @@ QUEUE=""
 
 ######################################### DO WORK ##########################################
 
-Template_list=( ${Template_list[@]} )
-Timepoint_list=( ${Timepoint_list[@]} )
-Subjlist=( ${Subjlist[@]} )
-
 for i in ${!Subjlist[@]}; do
```

Comment:
This variable was renamed, and this instance was missed.  However, `Subjects` versus `Subject` is less distinct than `Subjlist`, or we could expand that to `SubjectList` for readability.